print_list.js

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
Consider adding a Docstring/comment on the class. 
It seems to me that the class Node is part of the implementation of a linked list, it would be nice to not have to read all the code to get to know that.